docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -122,13 +122,15 @@ def merge_predictions_and_gt(predictions, old_ground_truth, field_config):
     return updated_predictions
 
 
-def overlap(line_1, line_2):
+def overlap(line_1, line_2, line_labels):
```

Comment:
I don't see `line_labels` being used in `filter_by_top1` - I suggest removing the argument from `filter_by_top1` and `overlap`.

Reply:
`filter_by_top1` does need the labels, but I have merged the regular labels and column_labels so that the function only takes one additional argument in addition to the predictions. Hopefully it is a bit easier to understand now.